Jenkinsfile

```diff
@@ -148,17 +148,17 @@ helpers.rootLinuxNode(env, {
                   integrate: {
                     // Build the client docker first so we can immediately kick off KBFS
                     def hasKBFSChanges = packagesToTest.keySet().findIndexOf { key -> key =~ /^github.com\/keybase\/client\/go\/kbfs/ } >= 0
-                    if (hasGoChanges && hasKBFSChanges) {
+                    if (true) {
```

Comment:
You can rebase to master and remove all of these `if (true)` segments since now Jenkinsfile changes trigger everything.

Reply:
nvm, realized the ldflags would trigger it anyway